packages/@ngtools/webpack/src/transformers/register_locale_data.spec.ts

```diff
@@ -0,0 +1,47 @@
+import * as ts from 'typescript';
+import { oneLine, stripIndent } from 'common-tags';
+import { transformTypescript } from './ast_helpers';
+import { registerLocaleData } from './register_locale_data';
+
+describe('@ngtools/webpack transformers', () => {
+  describe('register_locale_data', () => {
+    it('should add locale imports', () => {
+      const input = stripIndent`
+        import { enableProdMode } from '@angular/core';
+        import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+
+        import { AppModule } from './app/app.module';
+        import { environment } from './environments/environment';
+
+        if (environment.production) {
+          enableProdMode();
+        }
+
+        platformBrowserDynamic().bootstrapModule(AppModule);
+      `;
+      const output = stripIndent`
+        import __locale_fr__ from "@angular/common/locales/fr";
+        import { registerLocaleData as registerLocaleData } from "@angular/common";
```

Comment:
should be `import { registerLocaleData } from "@angular/common";`, I probably did something wrong in my transformer (not really wrong but useless)

Reply:
Fixed.